ESP_SHT3x/ESP_SHT3x.ino

```diff
@@ -32,14 +106,50 @@ void printShtMeasurments(float temp, float hum) {
     Serial.println("Failed to read humidity");
   }
 }
- 
-void loop() {
-  shtInit();
+
+// Sets relay pins to high or low depending on measurements
+void relayPower(int pin,float current, float limit) {
+  if(current < limit) {
+    digitalWrite(pin, LOW);
+  } else {
+    digitalWrite(pin, HIGH);
+  }
+}
+
+
+void shtAlgorithm() {
+  // Reads temperature and humidity from sht31 and stores it to appropriate variables
   float t = sht31.readTemperature();
   float h = sht31.readHumidity();
-
+  
+  // Displays current readings
   printShtMeasurments(t, h);
   
+  // Sets relay relevent pins to appropriate voltage levels
+  relayPower(humidPin, h, humidLimit);
+  relayPower(tempPin, t, tempLimit);
+}
+
+void DS18B20Algorithm() {
+  sensors.requestTemperatures(); 
+  float temperatureC = sensors.getTempCByIndex(0);
+
+  relayPower(tempPin, temperatureC, tempLimit);
```

Comment:
in addition to setting the heater on/off based on the readings, we'll want to set the humidifier based on a timer.

Reply:
moved to a new sprint